app/javascript/components/Users.js

```diff
@@ -0,0 +1,206 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import axios from "axios";
+import { daysInCompany, birthday, position } from "../utils/SortUsers";
+import SelectProject from "./SelectProject";
+import CurrentTime from "./CurrentTime";
+import profileImage from "../images/user-no-logo.png";
+
+class Users extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      filteredUsers: [],
+      usersList: []
+    };
+  }
+
+  componentDidMount() {
+    this.selectProject(this.props.usersPath);
+  }
+
+  addActiveClass = element => {
+    element.parentNode.childNodes.forEach(el => {
+      el.classList.remove("active");
+    });
+    element.classList.add("active");
+  };
+
+  filteredList = (search, currentUsers) =>
+    currentUsers.filter(listOfUsers =>
+      (
+        listOfUsers.email +
+        listOfUsers.phone +
+        listOfUsers.first_name +
+        listOfUsers.last_name +
+        listOfUsers.position
+      )
+        .toLowerCase()
+        .includes(search.toLowerCase())
+    );
+
+  sortBy = type => {
+    type === "position";
```

Comment:
Is it necessary?

Reply:
@alazarchuk equal by type? or sort by position?